features/facebookImport/test/importers/ad-view-language-importer.test.js

```diff
@@ -14,140 +14,94 @@ const datasets = [
-                    secondRelatedAccount,
-                ],
-            ];
+datasets.forEach(([language, dataset]) =>
+    describe(`Import ad views from ${language} dataset`, () => {
+        let result = null;
+        let relatedAccounts = null;
+        let firstRelatedAccount = null;
+        let secondRelatedAccount = null;
+
+        beforeAll(async () => {
+            const importingResult = await runAdsImportForDataset(dataset);
+            result = importingResult.result;
+            relatedAccounts = importingResult.relatedAccounts;
+            [firstRelatedAccount, secondRelatedAccount] = relatedAccounts.items;
         });
-    });
-});
-
-describe("Import ad views", () => {
-    console.log(testInputs);
-    test.each(testInputs)(
-        "returns success status for %s dataset.",
-        (language, result) => {
-            expectImportSuccess(result);
-        }
-    );
-
-    test.each(testInputs)(
-        "has two related accounts for %s dataset.",
-        (language, { relatedAccounts }) => {
-            expect(relatedAccounts.count).toBe(2);
-        }
-    );
-
-    test.each(testInputs)(
-        "has three ads for %s dataset.",
-        (language, { relatedAccounts }) => {
-            expect(relatedAccounts.adsCount).toBe(3);
-        }
-    );
-});
-*/
+
+        it("returns success status", () => expectImportSuccess(result));
+
+        it("has two related accounts", () =>
+            expect(relatedAccounts.count).toBe(2));
+
+        it("has three ads", () => expect(relatedAccounts.adsCount).toBe(3));
+
+        it("has four ad views", () =>
+            expect(relatedAccounts.adViewsCount).toBe(4));
+
+        it("has correct related account ids for first account", () => {
+            expect(firstRelatedAccount.rawId).toBeUndefined();
+            expect(firstRelatedAccount.urlId).toBe("companyx.com");
+        });
+
+        it("has correct related account ids for second account", () => {
+            expect(secondRelatedAccount.rawId).toBe("99888776543223");
+            expect(secondRelatedAccount.urlId).toBeUndefined();
+        });
+
+        it("has correct related accourl URLs", () => {
+            expect(firstRelatedAccount.url).toBe(
+                "https://www.facebook.com/companyx.com"
+            );
+            expect(secondRelatedAccount.url).toBe(
+                "https://www.facebook.com/99888776543223"
+            );
+        });
+
+        it("has correct related account names", () => {
+            expect(firstRelatedAccount.displayName).toBe("Company X");
+            expect(secondRelatedAccount.displayName).toBe("Company Y");
+        });
+
+        it("has correct ads count in accounts", () => {
+            expect(firstRelatedAccount.adsCount).toBe(2);
+            expect(secondRelatedAccount.adsCount).toBe(1);
+        });
+
+        it("has correct ad views count in accounts", () => {
+            expect(firstRelatedAccount.adViewsCount).toBe(3);
```

Comment:
And, OK, this was in the original test, but as long as you're changing it, it might be a good addition.

Reply:
This is also covered by AD_VIEWS_EXPECTED_DATA